pages/plugins.css

```diff
@@ -0,0 +1,4 @@
+.plugins {
+  display: flex;
+  flex-direction: column;
```

Comment:
Why did not bare blocks satisfy you?

Reply:
I feel like flex is more configurable. Seems like block would be sufficient too.